modules/auxiliary/scanner/http/cisco_ssl_vpn.rb

```diff
@@ -197,18 +224,7 @@ def do_login(user, pass, group)
 
         do_logout(resp.get_cookies)
 
-        report_hash = {
-          :host   => rhost,
-          :port   => rport,
-          :sname  => 'Cisco SSL VPN',
-          :user   => user,
-          :pass   => pass,
-          :group => group,
```

Comment:
I feel like the group is an important piece of information in order to login. I don't know what is the best way, but I think would worth to find a way of report it with `report_cred`